core/src/main/java/io/smallrye/context/impl/wrappers/ContextualSupplier2.java

```diff
@@ -29,14 +31,14 @@ public R get() {
     }
 
     @Override
-    public void captureThreadLocal(int index, ThreadLocal<Object> threadLocal, Object value) {
+    public void captureThreadScope(int index, ThreadScope<Object> ThreadScope, Object value) {
```

Comment:
```suggestion
    public void captureThreadScope(int index, ThreadScope<Object> threadScope, Object value) {
```

And the same for all the others